main.js

```diff
@@ -1,163 +1,198 @@
+        brand: 'Philips',
+        price: 689,
+        availableSizes: [55],
+        refreshRate: 100,
+        screenType: 'LED',
+        screenQuality: 'Ultra HD/4K',
+        smartTv: true,
+        options: {
+            wifi: true,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: true,
+        },
+        originalStock: 6,
+        sold: 3,
     },
-    originalStock: 6,
-    sold: 3,
-  },
-  {
-    type: 'B2450HD',
-    name: 'LED TV',
-    brand: 'Brandt',
-    price: 109,
-    availableSizes: [24],
-    refreshRate: 60,
-    screenType: 'LED',
-    screenQuality: 'Full HD',
-    smartTv: false,
-    options: {
-      wifi: false,
-      speech: false,
-      hdr: false,
-      bluetooth: false,
-      ambiLight: false,
+    {
+        type: 'B2450HD',
+        name: 'LED TV',
+        brand: 'Brandt',
+        price: 109,
+        availableSizes: [24],
+        refreshRate: 60,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: false,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
     },
-    originalStock: 10,
-    sold: 8,
-  },
-  {
-    type: '32WL1A63DG',
-    name: 'HD TV',
-    brand: 'Toshiba',
-    price: 161,
-    availableSizes: [32],
-    refreshRate: 50,
-    screenType: 'LED',
-    screenQuality: 'Full HD',
-    smartTv: false,
-    options: {
-      wifi: false,
-      speech: false,
-      hdr: true,
-      bluetooth: false,
-      ambiLight: false,
+    {
+        type: '32WL1A63DG',
+        name: 'HD TV',
+        brand: 'Toshiba',
+        price: 161,
+        availableSizes: [32],
+        refreshRate: 50,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
     },
-    originalStock: 10,
-    sold: 8,
-  },
-];
\ No newline at end of file
+];
+//Opdracht 1A
+const tvs = inventory.map((name) => {
+    return name.name
+})
+console.log(tvs)
+//----------------------------------------------------------------------------------------//
+//opdracht 1B
+const runOut = inventory.filter((sold) => {
+    return sold.originalStock < 6
+})
+//Ik wist niet welke methode goed zou zijn dus heb ik er 2 variante gemaakt.
+console.log(runOut)
+const runOut1 = inventory.map((sold) => {
+    return "Er zijn " + sold.sold + ' verkocht van de ' + sold.name +  ' en er waren er ' + sold.originalStock + ' in stock'
+})
+console.log(runOut1)
+//----------------------------------------------------------------------------------------//
+//opdracht 1C
+inventory.map((ambi) => {
+    if (ambi.options.ambiLight === true) {
```

Comment:
Dit zou je nog iets kunnen versimpelen
```suggestion
    if (ambi.options.ambiLight) {
```